core/templates/dev/head/pages/creator_dashboard/creator_dashboard.html

```diff
@@ -1126,9 +1117,8 @@ <h2>Suggestions to review</h2>
 
   <script src="/templates/dev/head/domain/creator_dashboard/CreatorDashboardBackendApiService.js"></script>
   <script src="/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js"></script>
-  <script src="/templates/dev/head/pages/suggestion_editor/ShowSuggestionModalForCreatorViewService.js"></script>
-  <script src="/templates/dev/head/pages/suggestion_editor/SuggestionModalService.js"></script>
-  <script src="/templates/dev/head/pages/suggestion_editor/ShowSuggestionModalForCreatorView.js"></script>
+  <script src="/templates/dev/head/pages/show-suggestion-editor-pages/show-suggestion-editor-pages-services/show-suggestion-modal-for-creator-view.service.js"></script>
+  <script src="/templates/dev/head/pages/show-suggestion-editor-pages/show-suggestion-editor-pages-services/suggestion-modal.service.js"></script>
```

Comment:
Why you were not adding this scripts in ``` base.html ``` and other files too?

Reply:
I thought that the base.html (and similarly others) already have a lot of scripts being loaded, and since loading them here wasn't breaking things, so I decided to load them here. 
Does this sound good?

Reply:
I think it's ok but we should fix this in all PRs - that is have the better method in all the modularity PRs.

What do you think @YashJipkate @import-keshav @seanlip @vojtechjelinek?

Reply:
Yep. This needs to be followed in all the PRs. Maybe we can ensure this in our reviews @ankita240796?

Reply:
Yes, sounds good.